domains/Bryan14Saenz.is-a.dev.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "is-a.dev",
+  "subdomain": "bryan14baenz", 
+  "owner": {
+    "email": "14bryansaenz@gmail.com"
+  },
+  "records": {
+    "CNAME": "https://portafoliob14s.netlify.app/"
```

`CNAME` record must contain a domain name only
```suggestion
    "CNAME": "portafoliob14s.netlify.app"
```